packages/fxa-auth-server/.vscode/launch.json

```diff
@@ -95,7 +95,8 @@
         "999999",
         "--colors",
         "${workspaceFolder}/test/local/payments/stripe.js",
-        "${workspaceFolder}/test/local/routes/subscriptions.js"
+        "${workspaceFolder}/test/local/routes/subscriptions.js",
+        "--exit"
```

Comment:
I assume at some point this task will be removed as its mainly useful in the specific case someone is adding test coverage just for this one portion of the auth-server vs. its entirety? Or should we have one task per logical unit of this? Or would it make sense to duplicate the task that runs the tests for the current file so that it can run coverage on the current file as well?

Reply:
I was thinking about updating it to run the current file.